src/main/java/Melchizedek.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class Melchizedek {
+
+    public static TaskList taskList = new TaskList();
+
+    public static void printHorizontalLine() {
```

Comment:
You could improve the naming of the function to something more specific, in the event you change the functionality of this.

You can consider renaming it to printParition or printBorder, in the event the border is change to vertical line or a line of + signs